pennylane/ops/qubit/parametric_ops.py

```diff
@@ -1269,6 +1269,182 @@ def control_wires(self):
         return self.wires[0:1]
 
 
+class PCPhase(Operation):
+    r"""
+    A Projector Controlled Phase gate.
+
+    .. math:: \Pi_{\phi}(\phi, dim=2, wires=[0,1]) = \begin{bmatrix}
+                e^{i\phi} & 0 & 0 & 0 \\
+                0 & e^{i\phi} & 0 & 0 \\
+                0 & 0 & 1 & 0 \\
+                0 & 0 & 0 & 1
+            \end{bmatrix}.
+
+    Args:
+        phi (float): rotation angle :math:`\phi`
+        dim (int): the qubit subspace on which we apply the phase shift
+        wires (Any, Wires): the wire the operation acts on
+        do_queue (bool): Indicates whether the operator should be
+            immediately pushed into the Operator queue (optional)
+        id (str or None): String representing the operation (optional)
+
+    **Details:**
+
+    * Number of wires: AnyWires
+    * Number of parameters: 1
+    * Number of dimensions per parameter: (0,)
+    * Gradient recipe: (~ ADD THIS ~)
+
+    **Example:**
+
+    >>> pc_op = qml.PCPhase(1.23, dim=3, wires=[1,2])
+    >>> qml.matrix(pc_op)
+    array([[0.33423773+0.9424888j, 0.        +0.j       , 0.        +0.j       , 0.        +0.j       ],
+           [0.        +0.j       , 0.33423773+0.9424888j, 0.        +0.j       , 0.        +0.j       ],
+           [0.        +0.j       , 0.        +0.j       , 0.33423773+0.9424888j, 0.        +0.j       ],
+           [0.        +0.j       , 0.        +0.j       , 0.        +0.j       , 1.        +0.j       ]])
```

Comment:
Where will the main example for QSVT live? When that exists, we might want to point users to look there for more info.

Reply:
Good question Tom! It will likely live in the main QSVT operation class doc-string. We can definitely link to it in the `see also` section once it is created.